turbo.json

```diff
@@ -25,7 +16,8 @@
   },
   "globalEnv": [
     "NEXT_PUBLIC_HOST",
+    "EXPRESS_PUBLIC_HOST",
```

Comment:
Maybe we can use it "PUBLIC_HOST" and replace NEXT_PUBLIC_HOST in all places, what do you think?

Reply:
the `NEXT_PUBLIC_` prefix is functional and allows the var to be used on the client side in nextjs apps

the `NEXT_PUBLIC_HOST` env variable is only used in the starter example. i think it's fine to add an express example-specific env var